### DISCUSSION TITLE
Consider using StaticWebAsset for serving app_plugin files in dev environment

### DISCUSSION THREAD

**Discussion by Nikkelmann** [leekelleher/umbraco-contentment#237] [Idea]
Other packages ([uSync](https://github.com/KevinJump/uSync/commit/a8782b693ce69df26a2528728f973dd80611b940) / [FullTextSearch](https://github.com/skttl/umbraco-fulltextsearch8/commit/b193ffe75c107da2181a91b179061708378339af)) seem to have shifted to this approch, which avoids having the files copied over during installation of the package.

I don't know if changing the sdk to `Microsoft.NET.Sdk.Razor` and adding `<StaticWebAssetBasePath>` is the only change needed.

**Comment by leekelleher** [comment]
Thanks for the suggestion @Nikkelmann.  It would have to be something that I'd look into with Contentment v5 (when I drop support for Umbraco v8). I'm unsure how my custom build steps (for bundling/minifying the CSS/JS) will work with a Razor Class Library, but I think uSync has a similar Gulp build task, so I would investigate how that works and attempt to do similar.

**Reply by leekelleher** [reply]
Good news, this will be coming in Contentment v5.
I already have a work-in-progress version in the [`dev/v5.x` branch](https://github.com/leekelleher/umbraco-contentment/tree/dev/v5.x).
